packages/terra-functional-testing/src/commands/viewport-helpers.js

```diff
@@ -0,0 +1,73 @@
+const Logger = require('../logger/logger');
+const terraViewports = require('../util/viewports');
+
+const logger = new Logger({ prefix: 'wdio-terra-service' });
+
+/**
+* Convenience method for getting the Terra test viewports by name.
+*
+* @param {string|string[]} sizes - the list of Terra viewport names.
+* @return {object} - Terra viewport sizes.
+*/
+const getViewports = (...sizes) => {
+  let viewportSizes = Object.keys(terraViewports);
+  if (sizes.length) {
+    viewportSizes = sizes;
+  }
+  return viewportSizes.map(formFactor => terraViewports[formFactor]);
+};
+
+/**
+* Sets the viewport for the test run to the Terra test viewport size.
+*
+* @param {string} [formFactor=huge] - the Terra test viewport.
+*/
```

Comment:
nit: Each of these comment blocks are slightly off on their indentation

Reply:
Updated: https://github.com/cerner/terra-toolkit/pull/438/commits/402ee523e9eabedfc323de69d40318f8b075dd7d